ddtrace/contrib/rq/__init__.py

```diff
@@ -78,13 +78,16 @@
 """
 import os
 
+import rq
```

Comment:
we cannot do this import here.

since this integration uses only `contrib/rq/__init__.py` and not the usual `required_modules` + `/patch.py` approach, we cannot import potentially unavailable modules here (hence why we do `import rq` in `def patch():`

Reply:
This is also why the `build_docs` job is failing.

Reply:
I figured this was why, didn't have much time to get back to it. I'll try splitting it to match the usual approach.